rclpy/src/rclpy/utils.cpp

```diff
@@ -104,20 +104,10 @@ get_rmw_implementation_identifier()
 }
 
 void
-assert_liveliness(py::capsule pyentity)
+assert_liveliness(rclpy::Publisher publisher)
```

Comment:
@ahcorde better not to pass by value:
```suggestion
assert_liveliness(rclpy::Publisher * publisher)
```